A few minor clarificaitons (#10)
w3c-fedid/Administration: proposals-CG-WG.md

  * What's needed from champions?
    * [ ] An [explainer](https://tag.w3.org/explainers/)
    * [ ] Alternatives and trade-offs considered
    * [ ] A specific preferred proposal, e.g., code samples, examples, or a spec PRs (if the formalism is needed to understand the proposal)
    * [ ] Browser and developer implementation experience (e.g., a prototype, dev trials, origin trials, etc) 
    * [ ] Confidence of developer demand and fitness for purpose (e.g. a developer that needs this proposal)
  * What's asked of the **Working Group**?
    * [ ] Working Group consensus to adopt the proposal as the basis for their work as a [Working Draft](https://www.w3.org/policies/process/#RecsWD).
    * [ ] Working Group identification of the list of (seemingly resolvable) issues that have to be addressed before [Stage 3](#stage-3).
    * [ ] An [Editor's Draft](https://www.w3.org/policies/process/#editors-draft) can be used to get PRs merged between [Working Draft](https://www.w3.org/policies/process/#RecsWD) revisions.
  
# Stage 3: Implementation

> NOTE: We don't yet have enough experience as we get towards Stage 3 and 4, so they aren't fully baked yet. It is possible that these

> > will change a lot as we learn, including potentially entirely removing them in favor of the W3C WG operation processs.

Reviewer: ```suggestion
> will change a lot as we learn, including potentially entirely replacing them with references to the W3C Process.
```
